src/components/ShareSaveButton/Button.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Button.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+export default function Button({ faIcon, label, onClick }) {
```

Comment:
where are we using the button? could you add it to a page so we can see it?

Reply:
I think it would be better to have a IconButton component maybe and pass it which icon we want instead of only save and share. What do you think?

Reply:
Good idea! I will make it an IconButton instead and import it inside the product page.